browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -0,0 +1,77 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+/*
+ * JSDialog.NavigatorPanel
+ */
+
+/* global app */
+class NavigatorPanel extends JSDialog.SidebarBase {
```

Comment:
I think "prefix" `JSDialog` is not needed in .ts files. We will detect class correctly.
It is needed only in JS files.